src/unix/notbsd/linux/other/mod.rs

```diff
@@ -548,6 +548,13 @@ pub const NLA_F_NESTED: ::c_int = 1 << 15;
 pub const NLA_F_NET_BYTEORDER: ::c_int = 1 << 14;
 pub const NLA_TYPE_MASK: ::c_int = !(NLA_F_NESTED | NLA_F_NET_BYTEORDER);
 
+#[repr(C)]
+pub enum idtype_t {
+    P_ALL,
+    P_PID,
+    P_PGID
+}
```

Comment:
Right now we actually tend to avoid literal `enum` types as they're not extensible (while the C side is), so perhaps these could just be `pub const P_ALL: c_int = i32`?